\chapterfont{\color{WongDarkBlue}}  % sets colour of chapter
\sectionfont{\color{WongDarkBlue}}  % sets colour of sections
\subsectionfont{\color{WongDarkBlue}}  % sets colour of subsections

\renewcommand\pcolor{WongDarkBlue}
\renewcommand{\headrule}{\hbox to\headwidth{%
		\color{WongDarkBlue}\leaders\hrule height \headrulewidth\hfill}} % color of title
\fancyfoot[LE,RO]{\thepage}

\chapter{Introduction}
\label{chap:introduction}

{ \Large \leftwatermark{
\put(-76.5,-75){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_wongDarkBlue.eps}}  \put(-67,-66.5){ {\color{white} 1 }}
\put(-67,-91.5){ 2 }
\put(-67,-116.5){ 3 }
\put(-67,-141.5){ 4 }
\put(-67,-166.5){ 5 }
\put(-67,-191.5){ 6 }
} \rightwatermark{
\put(346.5,-75){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_wongDarkBlue.eps}}  \put(350.5,-66.5){ {\color{white} 1 }}
\put(350.5,-91.5){ 2 }
\put(350.5,-116.5){ 3 }
\put(350.5,-141.5){ 4 }
\put(350.5,-166.5){ 5 }
\put(350.5,-191.5){ 6 }
}}

\newpage

\noindent

When the first draft of the human genome was released in June of 2000 the then president of the United States, Bill Clinton, told the press that the genome sequence would "revolutionize the diagnosis, prevention, and treatment of most, if not all, human diseases"\cite{collinsHasRevolutionArrived2010a}. Without a doubt, the first human genome sequence has revolutionized genomic research. Until that time mostly rare, monogenic diseases where studied with time-consuming methods, like linkage analysis and Sanger sequencing. After the first draft of the human genome was completed, there was a four-fold increase in causal genes identified for these rare, single-gene disorders\cite{claussnitzerBriefHistoryHuman2020b}. Additionally, computational and DNA sequencing technologies that were invented or optimized for this project\cite{hoodHumanGenomeProject2013} have been further developed and helped in bringing down the costs of DNA sequencing dramatically, from over \$2 billion for the first human genome to less than \$1000 now. This reduction in cost has made it possible for large projects like HapMap\cite{theinternationalhapmapconsortiumInternationalHapMapProject2003} and ENCODE\cite{consortiumENCODEENCyclopediaDNA2004} to produce maps of sequence variation and identify functional elements in the human genome. 

The impact of the project was and still is enormous and has allowed for many advancements within genetics and genomics research\cite{chicheBenchtobedsideReviewFulfilling2002a} and identification of disease genes and genetic diagnosis\cite{claussnitzerBriefHistoryHuman2020b}. However, it has only provided treatment in limited cases. Even though diagnosis of genetic disorders has improved, for the majority of cases genetic testing is still unable to find the causal genes and variants\cite{diemenRapidTargetedGenomics2017a}. Genetic tests are successfully used for preventative care of monogenic disorders, such as mastectomies or extra surveillance for BRCA1/2 carriers \cite{heemskerk-gerritsenSurvivalBilateralRiskreducing2019a}, but for complex disorders, where many variants have a small effect as opposed to one or a few variants having a large effect, this is still difficult\cite{claussnitzerBriefHistoryHuman2020b}. 

Despite the fact that the past 20 years has been full of incredible advances in genomics and genetics research, we are not yet close to the prevention and treatment of most human diseases. Still, there are many ways in which the field of genetics can assist in improving clinical outcome for rare and complex disorders. Currently, there are public-private partnerships under way that will help better in understanding of genotype-phenotype relationships\cite{HomepageInternationalCommon}, provide large data resources for researchers\cite{fingennFinnGenDocumentationR32020,sudlowUKBiobankOpen2015}, or assist in drug target identification\cite{carvalho-silvaOpenTargetsPlatform2019}. Improvements in polygenic risk scores will allow prediction of risk for polygenic traits\cite{natarajanpradeepPolygenicRiskScore2017, kheraGenomewidePolygenicScores2018}, which can help in preventative care for complex disorders. In drug development the drug compounds with direct genetic support increase substantially between preclinical and approved stage (2.0\% and 8.2\% respectively\cite{nelsonSupportHumanGenetic2015b}), showing the added benefit of integrating genetic research in the drug development pipeline. %Finally, integrating layers of omics and environmental data will address the missing heritability \cite{claussnitzerBriefHistoryHuman2020b}. 

In this thesis I address two important factors needed for using functional genomics to assist in improving healthcare. An insight into the effects of rare variants on gene expression and the genetic architecture of molecular traits, and the cell type and tissue context of the genetic effects on molecular traits.

\section{General introduction}
\subsection{Gene regulation through genetic variation}
The central dogma of molecular biology states that  genes, genetic information in DNA, is transcribed to RNA, which is then translated to a protein\cite{crickCentralDogmaMolecular1970}. Almost everything in cells is regulated by these proteins. They act as transcription factors (regulators of gene activity), antibodies (binding to specific foreign particles), enzymes (enablers of many chemical reactions), messenger, structural component or for transport or storage \cite{uzmanMolecularBiologyCell2003}. In addition, there are many genes of which the RNA transcript does not get translated to a protein. These non-coding genes are often involved in regulating the activity of genes \cite{shabalinaMammalianTranscriptomeFunction2004a}. 

The DNA that contains the genetic information consists of about 6.4 billion nucleotide bases divided over 23 chromosome pairs. When comparing a typical human genome sequence to the sequence of the reference human genome, about 4.1 to 5 million sites will be different (< 0.1\%). Most often, these differences, and the focus of this thesis, are either a difference of a single nucleotide or the insertion or deletion of a single base\cite{the1000genomesprojectconsortiumGlobalReferenceHuman2015}. Large deletions, duplications, inversions and other chromosome changes exist, but the focus of this thesis will be on single nucleotide changes and short insertions and deletions. Many of these differences will have no noticeable effect on any observable characteristics (the phenotype). Others will cause a difference in phenotype, for example in eye colour. Sometimes, these differences will be the cause of a genetic disorder, like cystic fibrosis or Huntingtonâ€™s disease, to give two examples. 

\subsection{The effects of single nucleotide polymorphisms}
Single Nucleotide Polymorphisms (SNPs), the difference of a single nucleotide base, can affect a gene in multiple ways. A SNP within a gene (coding region) can cause the protein product to be prematurely stopped\cite{deboeverMedicalRelevanceProteintruncating2018} or to fold differently\cite{vihinenTypesEffectsProtein2015}, reducing or completely removing its functionality. A SNP can also be located outside of the gene (non-coding region), for example in the start site of a gene or in an enhancer region, which can affect gene activity. A SNP at a transcription start site can cause a transcription factor to bind with less affinity, thereby lowering gene activity. This can also lead to disorders, such as the Pierre Robin sequence, which is caused by a changes in non-coding sequences around the SOX9 gene\cite{benkoHighlyConservedNoncoding2009}. However, most of the time variation in the non-coding region has a smaller effect than mutations in the coding region, and multiple non-coding mutations are necessary to cause a disorder.

\subsection{Genetic disorders}
Genetic disorders are often divided in two broad classes: monogenic disorders\cite{heidiRareGeneticDisorders2008} and complex disorders\cite{craigComplexDiseasesResearch2008}. Monogenic disorders are high effect variants where a SNP in a single gene is enough to cause a disease. Some well-known examples of monogenic disorders are cystic fibrosis\cite{keremIdentificationCysticFibrosis1989}, sickle-cell anaemia\cite{neelInheritanceSickleCell1949}, and Huntington's disease\cite{gusellaPolymorphicDNAMarker1983}. Complex disorders, on the other hand, are characterised by multiple variants, usually in regulatory regions, affecting the expression levels multiple genes. Each of these SNPs contributes a little to the risk of getting the disease. Examples of complex disorders include diabetes mellitus type 2\cite{americandiabetesassociationDiagnosisClassificationDiabetes2007}, Alzheimer's disease\cite{hardyAlzheimerDiseaseAmyloid1992}, and autism\cite{PolygenicTransmissionDisequilibrium}. For monogenic disorders it is relatively easy to identify the causal variant, as these variants are most often located within genes and either change important amino acids or cause frame shifts. Identifying causal variants for complex disorders is difficult, as each variant only has a small contribution, and environmental factors play an important role as well. 

One step towards identifying causal variants is by creating resources with genotype-phenotype relationships that can be easily used by many researchers.\cite{claussnitzerBriefHistoryHuman2020b}. Thanks to large collaborative efforts there are now biobanks with genotype data for more than 100.000 individuals (for example, UK biobank\cite{sudlowUKBiobankOpen2015}, FinnGenn\cite{fingennFinnGenDocumentationR32020}, and the Estonian Genome Project\cite{metspaluEstonianGenomeProject2004}). With these large datasets it is possible to identify those DNA sequence variants that are associated with increased or decreased risk for many different (disease) phenotypes, by comparing the genotypes of a large number of healthy people with that of a large group of people with a certain disease or other phenotype. These studies are called Genome Wide Association Studies (GWAS), and many of them are collected by the GWAS catalog\cite{bunielloNHGRIEBIGWASCatalog2019}. The 14 July 2020 GWAS Catalog release contains 116,480 variants for 4,467 of phenotypes. The question then remains, what are the molecular mechanisms in which these disease associated genes contribute to disease risk. One way is by mapping of quantitative trait loci\cite{membersofthecomplextraitconsortiumNatureIdentificationQuantitative2003} (QTLs). Quantitative traits can be any measurable phenotype that has variation due to genetics, such as height, or molecular traits such as gene expression, protein, or methylation levels. A QTL is a locus, a region on the DNA, where the variation affects the quantitative trait. With this, disease associated SNPs from GWAS studies can be linked to changes in for example expression levels, by mapping expression QTLs (eQTLs) for all GWAS SNPs.

%<add this somewhere maybe Natural Selection on Genes that Underlie Human Disease Susceptibility https://www-sciencedirect-com.proxy-ub.rug.nl/science/article/pii/S0960982208006015\cite{blekhmanNaturalSelectionGenes2008}>

\subsection{Human genetic evidence for drug development}
One area where genetics can be of definite help is in the pre-selection of candidate drug compounds. It is very costly to perform clinical trials, and even though there have been many technological advances in recent decades, drug development costs are high while there has been little increase in the output of new drugs \cite{cookLessonsLearnedFate2014a}. By including genetic information in the drug compound selection, the success rate in clinical development can be doubled\cite{nelsonSupportHumanGenetic2015b}. 

\section{This thesis}
The aim of this thesis is to get a better understanding of the context in which genetic risk factors influence gene expression levels. In part 1 we describe the effects of rare variants on gene expression in blood and the genetic architecture of molecular traits. In part 2 we identify the genetic regulation in context of different cell types in blood and in brain.

\subsection{Part 1 - architecture of molecular traits and rare variant signal}
In Chapter 2 we compared the heritability explained by all studies included in the GWAS catalog\cite{bunielloNHGRIEBIGWASCatalog2019} to the heritability that is explained by expression quantitative trait loci (eQTL) and methylation quantitative trait loci (meQTL) studies. We found that the genetic architecture of molecular traits is less polygenic than that of (disease) phenotypes. In chapter 3 we identified the disease categories for which allele-specific expression can be informative for prioritizing putative causal variants.

\subsection{Part 2 - cell type and tissue context of genetic regulation of gene expression}
In chapter 4 we developed two methods: 1. predicting the cell count proportions from whole blood samples using gene expression data, and 2. deconvoluting the cell type interacting eQTL effects found in whole blood samples. In chapter 5 we compared the genetic regulation of gene expression between brain and blood and identified which genes are affected by neurodegenerative and psychiatric risk factors. In chapter 6 I will discuss how these can contribute to improving health care outcomes.



\bibliographystyle{naturemag}
\bibliography{chapters/chapter1-introduction/chapter1-introduction}


% \cite{gibbsHumanGenomeProject2020} It should be noted that the hype was from front-facing... <- read this piece