\begin{appendices}

\chapter{Summary}
Our genetic code is stored in our DNA, which consists of four bases: adenine (A), cytosine (C) guanine (G) and cytosine (C). More than three billion of these bases are strung together in 23 chromosomes. Each of our cells has two copies of our genome, containing instructions that guide cellular processes such as growth, development, signalling, and many more, but can occasionally also be the basis of disease. 
A large part of the genetic instructions are contained in our genes, which are small regions in our genome which contain information to make ribonucleic acid molecules (RNA). The making of RNA is also called gene expression. For some genes, which we call protein-coding genes, RNA can be translated by ribosomes to protein molecules, and proteins perform most of the functions in our cell. For example, proteins are used to digest your food, to “communicate” between cells by sending and receiving signals, to create structures (e.g. your bones, skin, hair, etc), and have many other functions. Even the ribosome that translates your RNA to proteins is itself a protein. For other genes, which we call non-coding genes, the RNA does not get translated to proteins, but the RNA molecule itself is used for regulation. For many of these non-coding genes the function is not known, although some are involved in regulating gene expression.
Although genes contain the information for functional molecules that regulate most of our cellular functions, together they only make up about 1\% of our genome. The other 99\%, once thought of as “junk” DNA with no function, is now known to be involved in regulation of gene expression. Some examples of regions on the DNA that can regulate expression are regions that provide binding sites close to genes for the protein machinery that transcribes our DNA to RNA (promoter regions), provide protein  binding sites further away from genes that help to activate the transcription (enhancer regions) or to silence it (silencer regions), or protein binding sites that insulate genes (enhancer-blocker insulators) or prevent structural changes in the DNA (barrier insulators. 
Variation in our genomes can change the regulation of genes or the functionality of the protein products. Most often, genetic variation is benign, having no (detrimental) effect. However, in some cases it can be the basis of disease. For many diseases we know from previous studies which regions in the genome can increase risk of a disease. For example, we know that having the DNA base G on position 44,892,362 of chromosome 19 confers a slight increase in risk of getting Alzheimer’s disease. However, it is often not known why this might be. 
To get a better understanding of the molecular consequences of variations in the genome that might lead to disease, we want to know the downstream effects, and one such effect of interest is the change it can bring in the expression levels of genes.

For this thesis, I investigated how variation in our genomes affects gene regulation in blood and in the brain. I introduce the current state of genetics in chapter 1. <todo expand on chapter 1>. In chapter 2 we use existing studies to look at the genetic architecture of molecular traits. We find that single nucleotide polymorphisms (SNPs; a variation of a single base) can have much larger effects on molecular traits, such as gene expression, methylation of the DNA, or protein levels, than they have on (disease) phenotypes. This suggests that variation of a single gene due to disease associated SNPs has a limited effect on phenotype, and that many variants <todo finish sentence>. 
The previously published studies that we used focussed on common variants. We wanted to know the effect of rare variants on gene expression, variants that only one to five people out of a thousand have. In chapter 3 <todo> 

In the second part of this thesis, we look at genetic regulation of gene expression in different cellular contexts. Although each of our cells contain the same DNA, gene expression in different tissues, such as brain and blood, is quite different. Indeed, even between different cell types of the same tissue gene expression will differ. Therefore, genetic regulation of gene expression is likely also different between different cell types and tissues. In chapter 4 we look at differences in genetic regulation between different blood cell types. Because most gene expression is measured from samples that contain a mix of cell types (bulk samples) and the effect of specific cell types can’t directly be measured from these bulk samples, we developed a computational method to predict cell count proportions and subsequently use the cell count proportions to model genetic regulation in separate cell types. We show that this tool can accurately deconvolute which cell types contribute most to the genetic signal we measure in the bulk samples. In chapter 5 we re-use and combine data from 14 published brain datasets to create a map of genetic regulation of gene expression in the brain. We apply statistical methods to this data to identify genes that are likely causal for various neurological disorders. To give one example, we identified CYP24A1 to be putatively casual for multiple sclerosis, and using our method developed in chapter 4 we show that it likely acts in neurons. 



\chapter{Samenvatting}
Ons DNA bepaalt veel van onze kenmerken en ook onze gezondheid. 
\chapter{Acknowledgements}

One of the best parts of getting a PhD is meeting and working with many amazing people. I have been lucky to have had many great co-authors, colleagues and friends, who have made my time in Groningen really great.

Lude, I am vary grateful to have had you as my supervisor. Your enthusiasm for science is... And \\
Cisca, it has been great to have worked in your department, and I'm happy that I got to discuss many things during the PhD lunches.

Annique, thanks for being a great friend and colleague! I really enjoyed writing our paper in summer, and miss our spontaneous nights out. 

Joana thank you for being a great friend and colleague. 

Harm-Jan, it has been really fantastic to work with you on MetaBrain. I have learned a ton from you and it was really fun, too. I liked our little beatboxing when we were working in the meeting rooms. I admire your willingness to help 

R\'aul, I haven't thought about colour schemes before or since our deconvolution project, 

Freerk,

Adriaan,
Yang, 
Patrick
BIOS


Ellen, I miss our (bi)-weekly video calls! It has been really great collaborating with you, and thank you so much for making my visit to Boston smooth and fun. Heiko, thanks for your valuable inputs, and thank you for letting me visit your group, it was a really great experience. Jimmy, Chia-Yen, Paula

Iris
Zuzanna

I have been lucky to have (co)-supervised some really great students. Martijn, 

I would not have started doing a PhD without the mentors and teachers that I've had over the years. I would like to thank three specifically. Martijn, jou geweldige enthousiasme tijdens de intro dag was een grote reden dat ik met bioinformatica begonnen ben, <todo>. Sue, I got my first research experience with you <todo>. Francisco, I learned to be independent and <todo>.

Frits en Berdien, ik heb met veel plezier in het tuinhuisje gewoond

Thank you Mitchell and Mayra for 

Lieve papa en mama, bedankt voor jullie steun door de jaren heem

Anne-grete, minu kallis, thank you for always being there for me. You made finishing this book easier and less stressful. 

\chapter{About the author}

Niek Peter de Klein was born on the 9th of December, 1990, in Velp, the Netherlands. He completed his bachelor degree in Bioinformatics at the Hogeschool van Arnhem en Nijmegen, followed by a Masters degree in Bioinformatics at the Vrije Universiteit in Amsterdam. During his Bachelors and Masters degree, he got research experience in labs in Stanford, Dundee,Amsterdam, Luxembourg, Dundee. His research topics were on gene expression regulation and method development for mass spectrometry analysis.  

In 2015, Niek started his PhD at the Genetics Department of the University Medical Center in Groningen, the Netherlands in the group of prof. Lude Franke. His research focussed on genetic regulation of gene expression in different tissues. 


Currently, Niek is working as a postdoctoral researcher at the Wellcome Sanger Institute in Hinxton, UK. He is working in the labs of Dr. Emma Davenport and Dr. Gosia Trynka on identifying likely causal genes of systemic lupus erythematosus that might be used as future drug targets.

\cite{magnani2014comprehensive}
\cite{klein2015gene}

\renewcommand{\bibsection}{\section*{List of Publications}}

\bibliographystyle{naturemag}
\bibliography{non_chapters/list_of_publications/own-bib}


\end{appendices}
