\chapterfont{\color{Grey}}  % sets colour of chapter
\sectionfont{\color{Grey}}  % sets colour of sections
\subsectionfont{\color{Grey}}  % sets colour of subsections

\renewcommand\pcolor{Grey}
\renewcommand{\headrule}{\hbox to\headwidth{%
		\color{Grey}\leaders\hrule height \headrulewidth\hfill}} % color of title
\fancyfoot[LE,RO]{\thepage}

{ \Large \leftwatermark{
		\put(-67,-66.5){ 1 }
		\put(-67,-91.5){ 2 }
		\put(-67,-116.5){ 3 }
		\put(-67,-141.5){ 4 }
		\put(-67,-166.5){ 5 }
		\put(-76.5,-201){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_Grey.eps}} 
		\put(-67,-191.5){ {\color{white} 6 }}
	} \rightwatermark{
		\put(350.5,-66.5){ 1 }
		\put(350.5,-91.5){ 2 }
		\put(350.5,-116.5){ 3 }
		\put(350.5,-141.5){ 4 }
		\put(350.5,-166.5){ 5 }
		\put(346.5,-201){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_Grey.eps}} 
		\put(350.5,-191.5){ {\color{white} 6 }}
}}

\chapter{Discussion and Perspectives}
\chaptermark{}
\label{chap:discussion}



\newpage

\section{Summary of this thesis}
It is widely acknowledged that a major goal of most human genetics research is to improve outcomes in healthcare. This can be done in many different ways, for example by improving our understanding of health and disease\cite{mcguireRoadAheadGenetics2020, claussnitzerBriefHistoryHuman2020e}, which will help inform better diagnostics, or through biomarker identification and drug discovery. This thesis focuses on improving this understanding and studies how genetic variation is regulating gene expression. These ‘genotype‒expression maps’ help in identifying the downstream molecular mechanism of genetic variants and therefore provide insight into how these kinds of genetic risk factors cause disease. 

Genes operate in pathways and networks. These regulatory networks are incredibly complex and are regulated at many different layers. For many of the genetic variants that are associated to disease, it remains unclear how they disrupt these genes, pathways, or networks. However, knowledge on this is essential for understanding their role in disease and identifying potential drug targets. Through expression quantitative locus (eQTL) mapping, it is possible to determine the downstream gene expression effects of individual genetic variants. Through gene expression analyses, e.g. co-regulation analysis, it is subsequently possible to link individual genes together in regulatory networks. However, it has also become clear in the last few years that such these eQTL effects and networks are highly cell-type- and context-specific. As such, in order to gain meaningful insight for a certain disease of interest, it is important to study the cell types and tissues in which the disease is manifesting.

In this thesis, I have outlined the genetic architecture of molecular traits in \textbf{Chapter 1}\cite{claringbouldGeneticArchitectureMolecular2017} and identified rare genetic variants that affect gene expression levels in whole blood in \textbf{Chapter 2}\cite{kleinImbalancedExpressionPredicted2020}. I describe a statistical method that I developed to identify cell-type-specific eQTLs in \textbf{Chapter 4}\cite{raulaguirregamboaDeconvolutionBulkBlood2020} and use this method to analyse the effects of common variants on gene expression in the brain in \textbf{Chapter 5}.

In this discussion, I outline how these eQTL datasets and methods can be used to help prioritize candidate causal genes, assist in assessing the feasibility of drug compounds and create hypotheses for the role of identified genes on disease phenotypes, as well as considering how future work can improve on this.

\section{Understanding genetic regulation of gene expression}
Genome-wide association studies (GWAS) have revealed tens of thousands of genomic loci that confer disease risk. Thanks to the sharing of GWAS summary statistics, there are now hundreds of datasets of disease-associated variants\cite{visscher10YearsGWAS2017}. However, the mechanisms by which these variants increase disease risk, for example which genes are affected by the variant, are often not known.

\subsection{Understanding the effects of rare variants on gene expression}
As discussed in \textbf{chapter 2}, the effect sizes of established GWAS risk variants are usually higher for rare, low frequency variants. Because current eQTL and meQTL studies have substantially lower sample sizes than GWAS studies, it is difficult to assess the effects of rare genetic variants on gene expression and methylation levels. Generally, these studies can only be conducted for common variants (minor allele frequency (MAF) $>$ 0.05), because large numbers of samples are required to study low frequency variants, which is expensive to realize. However, it has been shown in yeast that rare variants can have strong effects on gene expression, often more than common variants\cite{bloomRareVariantsContribute2019}. A study of 472 genes in humans found that both very highly and very lowly expressed genes had a high burden of rare variants\cite{zhaoBurdenRareVariants2016}, showing that rare variants also have strong effects on gene expression in humans. Evidently, identifying the effects of rare variants on gene expression is important for understanding the genetic mechanism of disease. Others have found allele-specific expression (ASE) effects in many different tissues\cite{castelVastResourceAllelic2020}, and these effects can help to diagnose rare disease patients\cite{mohammadiGeneticRegulatoryVariation2019}. In \textbf{chapter 3}, we performed allele-specific expression in BIOS, a large healthy Dutch population cohort for which there are RNA-seq measurements from blood. Typically, when interested in rare variants, genotypes from whole genome sequencing (WGS) are preferable to genotypes from chip-array. Although it is possible to impute rare variants from chip genotypes, this is difficult without adding additional datasets to existing reference panels\cite{hoffmannStrategiesImputingAnalyzing2015}. Unfortunately, WGS data was only available for a subset of BIOS samples. To still be able to assess the effects of rare variants on gene expression, we used the RNA-seq data to call genotypes, as previously done by Deelen \emph{et al.}\cite{deelenCallingGenotypesPublic2015}. We also phased the RNA-seq‒based genotypes, which had not been done before with RNA-seq‒based genotypes. By phasing SNPs, it is possible determine which alleles are on the same haplotype, and this can improve ASE measurements since sequence read counts that overlap multiple SNPs can then be combined. We then used the WGS data of the subset of samples to validate both the genotypes and the phasing. Unfortunately, because SNPs can only be called using RNA-seq data for genes that are actually expressed, our phased haplotypes were often small. Because of this, we could get a better estimate of the allelic expression and combine counts from the same haplotype, but we were unable to phase variants outside of expressed genes and therefore could not assess the effects of regulatory regions on ASE.

Using this ASE approach, we were able to assess the effects of rare variants on gene expression levels. We found 5 individuals with predicted high-impact heterozygous variants in known autosomal dominant disease genes. When assessing the allelic imbalance of these SNPs, we found that the reference allele was always much more highly expressed than the alternative allele, and thanks to this feature, the gene expression levels were stable in these samples. One hypothesis of how this could happen is due to a positive feedback loop. Let's say there is a gene A and gene B. The expression of gene A is regulated by gene B, and higher expression of gene A causes gene B to downregulate gene A. In normal circumstances, both alleles of gene A produce a functional transcript and the expression of both alleles is about equal. However, if one of the alleles of gene A has, e.g., a stop-codon variant, the transcript from that allele will be degraded due to nonsense-mediated decay. In this case, there is less gene A product, and gene B therefore stops downregulating gene A. Gene A starts producing more transcripts, but due to nonsense-mediated decay, only one allele produces a transcriptional product. When gene A and gene B are in homeostasis, one of the alleles is much more highly expressed than the other. This scenario is in line with the fact that we see an enrichment in ASE for genes containing stop-coding variants.

\subsection{Cell and tissue context}
To find causal links and understand the variant mechanism, it is imperative to study effects in the right tissue and cell type. This was one of the main drivers for developing Decon2 in \textbf{chapter 4}, a tool that allows us to deconvolute cell type‒interacting eQTLs from bulk RNA-seq samples. Although we could show that our method works well, it has some limitations. Even though Decon2 compares favourably against the other cell count prediction tools xCell\cite{aranXCellDigitallyPortraying2017} and CIBERSORT\cite{newmanRobustEnumerationCell2015}, there is still an error in predicting cell count proportions based on gene expression. This error propagates when deconvoluting the cell type‒mediated eQTL effect from bulk tissue samples. Secondly, because we use cell count proportions, there are significant correlations between cell types. These correlations make it difficult to pinpoint the cell type‒specific effect. For example, if neutrophils and monocytes are anti-correlated and an interaction effect is found in neutrophils, it could also be that the effect is actually due to an opposite effect in monocytes. Thirdly, it is difficult to deconvolute the effect in specific sub-cell types, and, for Decon2, we deconvoluted only the six main immune cell types. However, there are many more sub-cell types that might be very relevant for understanding disease.

One method to overcome these three problems is to use single-cell RNA-seq\cite{tangMRNASeqWholetranscriptomeAnalysis2009}. Recently, this technique has been shown to be able to identify cell type‒specific eQTLs\cite{vanderwijstSinglecellRNASequencing2018b}. Cell types can be assigned to single cells on the basis of expression levels of marker genes, removing the error of predicting cell counts. Because eQTLs are calculated within only those cells from a specific cell type, there is no problem of correlated cell count proportions. eQTLs can also be assessed in even rare cell types as long as enough cells are available to have sufficient statistical power to detect eQTLs.

To further investigate the importance of cellular composition on eQTLs, we compared the \emph{cis}-eQTLs in different brain regions and in blood \emph{cis}-eQTLs in \textbf{chapter 4}. We observed that cortical and cerebral regions of the brain show highly concordant effects. More differences were observed when contrasting these brain regions with the cerebellum, something that was also observed in GTEx \cite{GTExConsortiumAtlas}. Differences in genetic regulation of gene expression in the brain cortex versus blood were often observed, although allelic concordance was high ($>$ 88\%).

\subsection{The need for more diversity in sampling}
One major obstacle of transitioning to precision medicine is that most of the large genetic cohort studies have been performed in European (52\%) or Asian (21\%) populations\cite{sirugoMissingDiversityHuman2019}. This is mainly problematic for polygenic disorders where the causal variants and tagging SNPs are in weak linkage disequilibrium\cite{sirugoMissingDiversityHuman2019}. For example, a multi-ethnic meta-analysis for pulmonary function identified more than 50 new loci when including individuals from African, Asian, and Hispanic/Latino ethnicities\cite{wyssMultiethnicMetaanalysisIdentifies2018}.  Additionally, certain SNPs may only be present in specific populations\cite{adalsteinsdottirberglindNationwideStudyHypertrophic2014}. In \textbf{chapter 5}, we had sufficient brain cortex samples from individuals from African and East Asian descent to perform eQTL analysis within these populations. Surprisingly, we observed very strong concordance and directionality of eQTL effects across European, African, and East Asian populations. This suggests that genetic effects on gene expression levels are generally shared across populations, but that some are population-specific due to differences in allele frequencies between populations.

%\subsection{Drug predictions}
%One application of genetic regulation of gene expression not addressed in this thesis is the prioritisation of drug compounds. One of the tools that can be used for drug discovery is the connectivity map\cite{lambConnectivityMapUsing2006}. It has measured gene expression levels in cell cultures that were treated by different drug compounds, and compared that to gene expression levels of cell cultures that were not treated by any drug compounds Fisetin

\section{Future work}
Looking back, the transformation of the genomics and genetics fields over the last 20 years has been amazing. There have been tremendous advances that have led to a much better understanding of the genetic basis of various diseases, and these insights are now often driving translational research. For example, for the drug development pipeline, the number drug mechanisms with direct genetic support increases from 2\% in the preclinical stage to 8.2\% for approved drugs\cite{nelsonSupportHumanGenetic2015}. Since we now know which variants are associated with increased risk of many diseases, thanks to the many large scale GWAS studies that have been done (currently the GWAS Catalog\cite{macarthurNewNHGRIEBICatalog2017} contains 4118 studies for 3418 traits), this information can be leveraged to improve drug development. However, to do this optimally, we need to understand the mechanism by which these variants drive disease risk and to link them to causal genes and proteins.

With continued efforts in building comprehensive maps of genotype‒phenotype relationships under various circumstances, further advancements in high-throughput functional variant screenings such as CRISPR-Cas, and continued application of new insights from these efforts for new therapeutic options, we will see genetics play more and more important roles in healthcare and drug development.

Altogether, future studies aiming to improve our understanding of molecular effects of genetics need to combine many different aspects. Ideally, they should be done with multi-ethnic individuals. If interested in a specific disease, this can be limited to populations with high prevalence. These studies should be conducted using single-cell technologies, as the cellular context of genetic effects is very important. As large\cite{vosaUnravelingPolygenicArchitecture2018} and multi-tissue\cite{GTExConsortiumAtlas} studies having already been completed with bulk data, findings from single-cell studies can be replicated in bulk tissues if needed. Since physiology also plays an important role, the most informative studies would include both cases and healthy individuals. In case of a genetic disorder that manifests when cells are stimulated, for example with immune disorder where immune cells that are stimulated by allergens are overactive, stimulations should also be included to capture the relevant gene regulation. Finally, although this thesis focuses on genetic effects on gene expression, gene expression alone will not be able to capture the full cellular response. Gene expression and protein levels are reasonably but not perfectly correlated\cite{buccitelliMRNAsProteinsEmerging2020}. Additionally, SNPs might have an effect on chromatin state or methylation but not on gene expression directly. Therefore, measurements should include not only RNA-seq, but also proteomics, ATAC-seq, or DNase-seq. 

\subsection{Collaboration and open data}
Finally, much of the work in this thesis, and all of \text{chapter 5}, is built on publicly available data. It has to be highlighted that without the academic community being so willing to sharing their data, much of the genetic work discussed here would not have been possible. In this context, it is encouraging to see so many large-scale academic collaborations, such as eQTLgen\cite{vosaUnravelingPolygenicArchitecture2018} and GTEx\cite{GTExConsortiumAtlas}, pre-competitive public-private collaborations, such as OpenTargets, the UKBiobank\cite{sudlowUKBiobankOpen2015} and Fingenn\cite{fingennFinnGenDocumentationR32020}, as well as large disease-specific collaborations like Psychencode\cite{DbGaPStudy,ngXQTLMapIntegrates2017,wangComprehensiveFunctionalGenomic2018} and AMP-AD\cite{hodesAcceleratingMedicinesPartnership2016}.


\bibliographystyle{naturemag}
\bibliography{chapters/chapter6-discussion/chapter6-discussion}
