\chapter{Discussion and Perspectives}
\label{chap:discussion}

{ \Large \leftwatermark{
\put(-67,-66.5){ 1 }
\put(-67,-91.5){ 2 }
\put(-67,-116.5){ 3 }
\put(-67,-141.5){ 4 }
\put(-67,-166.5){ 5 }
\put(-67,-191.5){ 6 }
\put(-67,-216.5){ 7 }
\put(-76.5,-250){\includegraphics[scale=0.8]{img/thumbindex.eps}} \put(-67,-241.5){ {\color{white} 8 }}
} \rightwatermark{
\put(350.5,-66.5){ 1 }
\put(350.5,-91.5){ 2 }
\put(350.5,-116.5){ 3 }
\put(350.5,-141.5){ 4 }
\put(350.5,-166.5){ 5 }
\put(350.5,-191.5){ 6 }
\put(350.5,-216.5){ 7 }
\put(346.5,-250){\includegraphics[scale=0.8]{img/thumbindex.eps}} \put(350.5,-241.5){ {\color{white} 8 }}
}}

\newpage

\section{Discussion}

Summary of this thesis
The regulatory networks in which genes form pathways are incredibly complex, with many layers of regulation. When trying to understand the function of a gene or the potential of a gene to be a candidate drug target, it is important to know how this gene functions within these complicated pathways. One way to approach this is by identifying which genetic factors affect the genes through eQTL studies, and subsequently building gene correlation networks to identify which pathways these genes are involved in. Preferably this should be done in cell type or tissue that is relevant for the phenotype of interest. In this thesis we have outlined the genetic architecture of molecular traits and identified the effect of rare variants on gene expression data in whole blood. We have developed statistical methods to look at cell type interacting genetic effects in blood, and we have analysed the effects of common variants on gene expression in the brain. I will now discuss how this data and these methods can be used to help prioritize candidate causal gene prioritization, assist in assessing the feasibility of drug compounds and create hypotheses for the role of identified genes on disease phenotypes. Finally, I will give a perspective of what the future direction of this work could be. 

\section{Future perspective}

