\chapterfont{\huge\color{Grey}}  % sets colour of chapter
\sectionfont{\color{Grey}}  % sets colour of sections
\subsectionfont{\color{Grey}}  % sets colour of subsections
\renewcommand\pcolor{Grey}

\renewcommand{\headrule}{\hbox to\headwidth{%
		\color{Grey}\leaders\hrule height \headrulewidth\hfill}} % color of title
\fancyfoot[LE,RO]{\thepage}


{ \Large \leftwatermark{
		\put(-67,-66.5){ 1 }
		\put(-67,-91.5){ 2 }
		\put(-67,-116.5){ 3 }
		\put(-67,-141.5){ 4 }
		\put(-67,-166.5){ 5 }
		\put(-76.5,-201){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_Grey.eps}} 
		\put(-67,-191.5){ {\color{white} 6 }}
	} \rightwatermark{
		\put(350.5,-66.5){ 1 }
		\put(350.5,-91.5){ 2 }
		\put(350.5,-116.5){ 3 }
		\put(350.5,-141.5){ 4 }
		\put(350.5,-166.5){ 5 }
		\put(346.5,-201){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_Grey.eps}} 
		\put(350.5,-191.5){ {\color{white} 6 }}
}}

\cleardoublepage
\makeatletter
\let\savedchap\@makechapterhead
\def\@makechapterhead{\vspace*{-3cm}\savedchap}
\chapter{Discussion and Perspectives}
\chaptermark{}
\let\@makechapterhead\savedchap
\makeatletter
\chaptermark{}

\label{chap:discussion}




\newpage


\Needspace{10\baselineskip}
\section{Summary of this thesis}

It is widely acknowledged that a major goal of most human genetics research is to improve outcomes in healthcare. This can be done in many different ways, for example by improving our understanding of health and disease\cite{mcguireRoadAheadGenetics2020, claussnitzerBriefHistoryHuman2020}, which will help inform better diagnostics, or through biomarker identification and drug discovery. This thesis focuses on improving this understanding and studies how genetic variation is regulating gene expression. These ‘genotype‒expression maps’ help in identifying the downstream molecular mechanism of genetic variants and therefore provide insight into how these kinds of genetic risk factors cause disease. 


Genes operate in pathways and networks. These regulatory networks are incredibly complex and are regulated at many different layers. For many of the genetic variants that are associated to disease, it remains unclear how they disrupt these genes, pathways, or networks. However, knowledge on this is essential for understanding their role in disease and identifying potential drug targets. Through expression quantitative locus (eQTL) mapping, it is possible to determine the downstream gene expression effects of individual genetic variants. Through gene expression analyses, e.g. co-regulation analysis, it is subsequently possible to link individual genes together in regulatory networks. However, it has also become clear in the last few years that such these eQTL effects and networks are highly cell-type- and context-specific. As such, in order to gain meaningful insight for a certain disease of interest, it is important to study the cell types and tissues in which the disease is manifesting.


In this thesis, I have outlined the genetic architecture of molecular traits in \textbf{Chapter 1}\cite{claringbouldGeneticArchitectureMolecular2017} and identified rare genetic variants that affect gene expression levels in whole blood in \textbf{Chapter 2}\cite{kleinImbalancedExpressionPredicted2020}. I describe a statistical method that I developed to identify cell-type-specific eQTLs in \textbf{Chapter 4}\cite{raulaguirregamboaDeconvolutionBulkBlood2020} and use this method to analyse the effects of common variants on gene expression in the brain in \textbf{Chapter 5}.


In this discussion, I outline how these eQTL datasets and methods can be used to help prioritize candidate causal genes, assist in assessing the feasibility of drug compounds and create hypotheses for the role of identified genes on disease phenotypes, as well as considering how future work can improve on this.


\Needspace{10\baselineskip}
\section{Understanding genetic regulation of gene expression}

Genome-wide association studies (GWAS) have revealed tens of thousands of genomic loci that confer disease risk. Thanks to the sharing of GWAS summary statistics, there are now hundreds of datasets of disease-associated variants\cite{visscher10YearsGWAS2017}. However, the mechanisms by which these variants increase disease risk, for example which genes are affected by the variant, are often not known.


\Needspace{10\baselineskip}
\subsection{Understanding the effects of rare variants on gene expression}

As discussed in \textbf{chapter 2}, the effect sizes of established GWAS risk variants are usually higher for rare, low frequency variants. Because current eQTL and meQTL studies have substantially lower sample sizes than GWAS studies, it is difficult to assess the effects of rare genetic variants on gene expression and methylation levels. Generally, these studies can only be conducted for common variants (minor allele frequency (MAF) $>$ 0.05), because large numbers of samples are required to study low frequency variants, which is expensive to realize. However, it has been shown in yeast that rare variants can have strong effects on gene expression, often more than common variants\cite{bloomRareVariantsContribute2019}. A study of 472 genes in humans found that both very highly and very lowly expressed genes had a high burden of rare variants\cite{zhaoBurdenRareVariants2016}, showing that rare variants also have strong effects on gene expression in humans. Evidently, identifying the effects of rare variants on gene expression is important for understanding the genetic mechanism of disease. Others have found allele-specific expression (ASE) effects in many different tissues\cite{castelVastResourceAllelic2020}, and these effects can help to diagnose rare disease patients\cite{mohammadiGeneticRegulatoryVariation2019}. In \textbf{chapter 3}, we performed allele-specific expression in BIOS, a large healthy Dutch population cohort for which RNA-seq gene expression measurements are available from blood. Typically, when interested in rare variants, genotypes from whole genome sequencing (WGS) are more useful than imputed rare genotypes that have been derived from genotyping arrays: although it is sometimes possible to impute rare variants from chip genotypes, this is difficult for very rare variants, particulary when  not adding additional whole-genome sequence datasets to existing reference panels\cite{hoffmannStrategiesImputingAnalyzing2015}. This also applies to the BIOS cohort: for only a small subset of these samples whole-genome sequence data was available. In order to still be able to assess the effects of rare variants on gene expression, we used the RNA-seq data it self to call genotypes, as was previously done by Deelen \textit{et al.}\cite{deelenCallingGenotypesPublic2015}. We also phased the RNA-seq‒based genotypes, which had not been done before with RNA-seq‒based genotypes. By phasing SNPs, it is possible to determine which alleles are on the same haplotype, and this can improve ASE measurements since sequence read counts that overlap multiple SNPs can then be combined. We then used the WGS data of the subset of samples to validate both the genotypes and the phasing. Unfortunately, because SNPs can only be called using RNA-seq data for genes that are actually expressed, our phased haplotypes were often small. Because of this, we could get a better estimate of the allelic expression and combine counts from the same haplotype, but we were unable to phase variants outside of expressed genes and therefore could not assess the effects of regulatory regions on ASE.


Using this ASE approach, we were able to assess the effects of rare variants on gene expression levels. We found 5 individuals with predicted high-impact heterozygous variants in known autosomal dominant disease genes. When assessing the allelic imbalance of these SNPs, we found that the reference allele was always much more highly expressed than the alternative allele, and thanks to this feature, the gene expression levels were stable in these samples. One hypothesis of how this could happen is due to a positive feedback loop. This can be best explained through a hypothetical example: For instance, let us assume there is a gene A and gene B. The expression of gene A is regulated by gene B, and higher expression of gene A causes gene B to downregulate gene A. In normal circumstances, both alleles of gene A produce a functional transcript and the expression of both alleles is about equal. However, if one of the alleles of gene A has, e.g., a stop-codon variant, the transcript from that allele will be degraded due to nonsense-mediated decay. In this case, there is less gene A product, and gene B therefore stops downregulating gene A. Gene A starts producing more transcripts, but due to nonsense-mediated decay, only one allele produces a transcriptional product. When gene A and gene B are in homeostasis, one of the alleles is much more highly expressed than the other. This scenario is in line with the fact that we see an enrichment in ASE for genes containing stop-coding variants.


\Needspace{10\baselineskip}
\subsection{Tissue context}
To investigate the importance of sampling the right tissues for eQTLs, we compared the \textit{cis}-eQTLs in different brain regions and in blood \textit{cis}-eQTLs in \textbf{chapter 4}. We observed that cortical and cerebral regions of the brain show highly concordant effects. Larger differences were observed when contrasting these brain regions with the cerebellum, something that was also observed in GTEx \cite{thegtexconsortiumGTExConsortiumAtlas2020}. Differences in genetic regulation of gene expression in the brain cortex versus blood were often observed, although allelic concordance was high ($>$ 88\%). 

Often, blood based eQTL datasets are used in downstream analysis of studies in other tissues, for example in the brain\cite{wrayGenomewideAssociationAnalyses2018, qiIdentifyingGeneTargets2018a} or pancreas \cite{keEvaluationPolymorphismsMicroRNA2020,inshawGeneticVariantsPredisposing2020}, because the largest eQTL dataset (eQTLgen\cite{vosaUnravelingPolygenicArchitecture2018}) is made using blood samples. However, we find that 24\% of shared eQTLs between our brain eQTLs and eQTLgen \cite{vosaUnravelingPolygenicArchitecture2018} showed opposite allelic effects. Although in the previous mentioned studies eQTLgen is used in conjunction with tissue specific datasets, such as GTEx\cite{thegtexconsortiumGTExConsortiumAtlas2020} (44 tissues) and ROSMAP (brain), these datasets have small sample numbers, and much of their downstream analysis will be driven by the blood eQTLs. Although we have shown that the cerebral and cortical region of the brain is quite different from other tissues (\textbf{Chapter 5, Figure 3E}) and other tissues might share direction of effect for more than 76\% of the \textit{cis}-eQTLs, it is important to get large eQTL datasets for all tissues to improve downstream analysis. This is further highlighted by the comparison in \textbf{chapter 4} of likely causal genes that we have identified for Multiple Sclerosis (MS) using Mendelian Randomization (MR) with both the MetaBrain eQTLs (from cortex) and eQTLgen eQTLs (from blood). We identified 3 likely causal MS genes (\textit{SLC12A5}, \textit{CCDC155}, and \textit{MYNN}) which were not found using the larger eQTLgen blood dataset, even though the eQTLgen dataset is four times larger than the MetaBrain dataset. This shows the importance of creating large eQTL datasets in all tissues, so that disease relevant effects can be identified also when present in a specific tissues.

\Needspace{10\baselineskip}
\subsection{Cell context}

A large part of the reason that different tissues have different genetic effects is due to the difference in cell type composition between tissues. Lymphocytes and neutrophils in blood will have different cellular processes than neurons and oligodendrocytes in the brain. However, lymphocytes in the blood will also be different from neutrophils in the blood. It is therefore imperative to study genetic effects in cell types as well. This can be done by purifying cell types\cite{adamsBLUEPRINTDecodeEpigenetic2012} or by using single-cell RNA-seq\cite{tangMRNASeqWholetranscriptomeAnalysis2009}, which has been shown to work for eQTL analyis as well\cite{wijstSinglecellRNASequencing2018}. However, these methods are still either expensive and laborious (purifying cells) or relatively novel (single-cell RNA-seq) and therefore there are not yet single cell RNA-seq datasets with large sample sizes available. Instead, to make use of the already existing RNA-seq data, we developed Decon2 in \textbf{chapter 4}, a tool that allows us to deconvolute cell type‒interacting eQTLs from bulk RNA-seq samples. We applied this method to the brain samples of \textbf{chapter 5}. This allowed us to link two likely causal MS genes (found using MR and colocalization) to their most likely relevant cell type: \textit{CYP24A1} in neurons and \textit{CLECL1} in microglia. One factor that makes functional validation of GWAS variants challenging is that often the causal cell type is not known\cite{canogamezGWASFunctionUsing2020} and being able to prioritize likely causal cell types is a great aid for functional follow-up.  


\Needspace{10\baselineskip}
\subsection{The need for more diversity in sampling}

One major obstacle of transitioning to precision medicine is that most of the large genetic cohort studies have been performed in European (52\%) or Asian (21\%) populations\cite{sirugoMissingDiversityHuman2019}. This is mainly problematic for polygenic disorders where the causal variants and tagging SNPs are in weak linkage disequilibrium\cite{sirugoMissingDiversityHuman2019}. For example, a multi-ethnic meta-analysis for pulmonary function identified more than 50 new loci when including individuals from African, Asian, and Hispanic/Latino ethnicities\cite{wyssMultiethnicMetaanalysisIdentifies2018}.  Additionally, certain SNPs may only be present in specific populations\cite{adalsteinsdottirberglindNationwideStudyHypertrophic2014}. In \textbf{chapter 5}, we had sufficient brain cortex samples from individuals from African and East Asian descent to perform eQTL analysis within these populations. Surprisingly, we observed very strong concordance and directionality of eQTL effects across European, African, and East Asian populations. This suggests that genetic effects on gene expression levels are generally shared across populations, but that some are population-specific due to differences in allele frequencies between populations. 


\Needspace{10\baselineskip}
\section{Future perspectives}

Looking back, the transformation of the genomics and genetics fields over the last 20 years has been amazing. There have been tremendous advances that have led to a much better understanding of the genetic basis of various diseases, and these insights are now often driving translational research. For example, for the drug development pipeline, the number of drug mechanisms with direct genetic support has increased from 2\% in the preclinical stage to 8.2\% for approved drugs\cite{nelsonSupportHumanGenetic2015}. Since we now know which variants are associated with increased risk of many diseases, thanks to the many large scale GWAS studies that have been done (currently the GWAS Catalog\cite{macarthurNewNHGRIEBICatalog2017} contains 4118 studies for 3418 traits), this information can be leveraged to improve drug development. However, to do this optimally, we need to understand the mechanism by which these variants drive disease risk and to link them to causal genes and proteins. There are several ways that we can do this in the future.

\Needspace{10\baselineskip}
\subsection{Tissue and cell type effects}
Having data available for many human tissues through GTEx\cite{thegtexconsortiumGTExConsortiumAtlas2020} has been very important for studying genetics effects in many different tissues. However, as we have shown in \textbf{chapter 5}, much larger sample sizes are necessary if one is interested in \textit{trans}-eQTLs or in downstream statistical methods to identify likely causal genes using for instance Mendelian Randomization. It is therefore important to keep increasing sample sizes for genetic studies. It might be difficult to justify the cost of large scale eQTL studies just to increase samples size. However, as in this thesis I have shown that meta-analysing datasets with individuals from different ethnicities and disease status is effective in increasing statistical power to perform trans-eQTL and colocalization analyses, we can continue by generating smaller, disease or ethnicity specific datasets. Including patient samples is important either way, as we have shown in \textbf{chapter 5}, where we identified a \textit{trans}-eQTL SNP that is associated with neuron proportions, which were lower in the Alzheimer patients included in our meta-analysis. 

I have previously discussed our computational approach to deconvolute cell type specific genetic effects from RNA-seq samples that are mixtures of cell types (such as bulk blood or brain samples). However, as outlined in \textbf{chapter 4}, there are significant limitations to this approach. The three major limitations are that the error in the prediction of cell counts propagates to the eQTL deconvolution, correlations between cell counts of different cell types make it difficult to pinpoint cell type specific effects, and deconvolution of rare cell types is difficult) These limitations can be overcome by using single-cell RNA-seq\cite{tangMRNASeqWholetranscriptomeAnalysis2009}. This technique has been shown to be able to identify cell type‒specific eQTLs\cite{wijstSinglecellRNASequencing2018}. Cell types can be assigned to single cells on the basis of expression levels of marker genes, removing the error of predicting cell counts. Because eQTLs are calculated within only those cells from a specific cell type, there is no problem of correlated cell count proportions. eQTLs can also be assessed in even rare cell types as long as enough cells are available to have sufficient statistical power to detect eQTLs, or by enriching the rare cell types.


\Needspace{10\baselineskip}
\subsection{Functional validation}

In this thesis we have identified variants that are regulating gene expression and have linked these to disease phenotypes. For neurological related phenotypes, we have identified genes that are likely causal for the disease. However, we have not functionally validated these in the lab. The genes that we identified are excellent targets for CRISPR-CAS9 knockout screening\cite{agrotisNewAgeFunctional2015}. Since we have statistically determined which cell types are likely the main contributors to the genetic signal, this can be tested in the most likely relevant cell type. 

\Needspace{10\baselineskip}
\subsection{Future study design}
Altogether, future studies aiming to improve our understanding of molecular effects of genetics need to combine many different aspects. Ideally, they should be done with multi-ethnic individuals. When interested in a specific disease, this can be limited to populations with high prevalence. These studies should be conducted using single-cell technologies, as the cellular context of genetic effects is very important. As large\cite{vosaUnravelingPolygenicArchitecture2018} and multi-tissue\cite{thegtexconsortiumGTExConsortiumAtlas2020} studies having already been completed with bulk data, findings from single-cell studies can be replicated in bulk tissues if needed. Since physiology also plays an important role, the most informative studies would include both cases and healthy individuals. In case of a genetic disorder that manifests when cells are stimulated, for example with immune disorder where immune cells that are stimulated by allergens are overactive, stimulations should also be included to capture the relevant gene regulation. Finally, although this thesis focuses on genetic effects on gene expression, gene expression alone will not be able to capture the full cellular response. Gene expression and protein levels are reasonably but not perfectly correlated\cite{buccitelliMRNAsProteinsEmerging2020}. Additionally, SNPs might have an effect on chromatin state or methylation but not on gene expression directly. Therefore, measurements should include not only RNA-seq, but also proteomics, ATAC-seq, or DNase-seq. 

\Needspace{10\baselineskip}
\section{Conclusions}
The work in this thesis has improved our understanding of the effects of common and rare genetic variants on gene expression in blood and brain. We have developed a statistical method to determine which cell types contribute to genetic effects we find in mixtures of cells. We have created a large brain eQTL dataset and shown how to use this dataset to identify likely causal genes, with a focus on multiple sclerosis and ALS. Future work can improve on this by increasing sample sizes for additional tissues and by using single-cell experiments to directory measure cell type specific effects. 

\bibliographystyle{naturemag}

\bibliography{chapters/chapter6-discussion/chapter6-discussion}

