\chapterfont{\color{Grey}}  % sets colour of chapter
\sectionfont{\color{Grey}}  % sets colour of sections
\subsectionfont{\color{Grey}}  % sets colour of subsections

\renewcommand\pcolor{Grey}
\renewcommand{\headrule}{\hbox to\headwidth{%
		\color{Grey}\leaders\hrule height \headrulewidth\hfill}} % color of title
\fancyfoot[LE,RO]{\thepage}

{ \Large \leftwatermark{
		\put(-67,-66.5){ 1 }
		\put(-67,-91.5){ 2 }
		\put(-67,-116.5){ 3 }
		\put(-67,-141.5){ 4 }
		\put(-67,-166.5){ 5 }
		\put(-76.5,-201){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_Grey.eps}} 
		\put(-67,-191.5){ {\color{white} 6 }}
	} \rightwatermark{
		\put(350.5,-66.5){ 1 }
		\put(350.5,-91.5){ 2 }
		\put(350.5,-116.5){ 3 }
		\put(350.5,-141.5){ 4 }
		\put(350.5,-166.5){ 5 }
		\put(346.5,-201){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_Grey.eps}} 
		\put(350.5,-191.5){ {\color{white} 6 }}
}}

\chapter{Discussion and Perspectives}
\chaptermark{}
\label{chap:discussion}



\newpage

\section{Summary of this thesis}
It is widely acknowledged that a major goal of most (human) genetics research is to improve outcomes in healthcare. This can be done in many different ways, by improving our understanding of health and disease\cite{mcguireRoadAheadGenetics2020, claussnitzerBriefHistoryHuman2020e}, which will help inform better diagnostics, biomarker identification and drug discovery. This thesis focuses on improving such understanding, and studies how genetic variation is regulating gene expression. Such ‘genotype-expression maps’ help identifying the downstream molecular mechanism of genetic variants and therefore provides insight how such genetic risk factors cause disease. 

Genes operate in pathways and networks. These regulatory networks are incredibly complex and are regulated at many different layers. For many genetic variants that are associated to disease it remains unclear how they disrupt these genes, pathways and networks. Knowledge on this is essential in order to understand their role in disease and to identify potential drug targets. Through expression quantitative locus (eQTL) mapping it is possible to determine the downstream gene expression effects of individual genetic variants. And through gene expression analyses, e.g. co-regulation analysis, it is subsequently possible to link individual genes together in regulatory networks. However, it has become clear in the last few years that such these eQTL effects and networks are highly cell-type and also context-specific. As such, in order to gain meaningful insight for a certain disease of interest it is important to study cell-types and tissues in which this disease is manifesting itself.

In this thesis I have outlined the genetic architecture of molecular traits in \textbf{Chapter 1}\cite{claringbouldGeneticArchitectureMolecular2017} and identified rare genetic variants that affect gene expression levels in whole blood in \textbf{Chapter 2}\cite{kleinImbalancedExpressionPredicted2020}. I describe a statistical method that I developed to identify cell-type specific eQTLs in \textbf{Chapter 4}\cite{raulaguirre-gamboaDeconvolutionBulkBlood2020}, and I analysed the effects of common variants on gene expression in the brain in \textbf{Chapter 5}, where I applied the method from chapter 4.

In this discussion, I outline how this eQTL datasets and methods can be used to help prioritize candidate causal genes, assist in assessing the feasibility of drug compounds and create hypotheses for the role of identified genes on disease phenotypes, and how future work can improve on this.

\section{Understanding genetic regulation of gene expression}
Genome-wide association studies (GWAS) have revealed tens of thousands of genomic loci that confer disease risk. Thanks to the sharing of GWAS summary statistics there are now hundreds of datasets with disease associated variants available\cite{visscher10YearsGWAS2017}. However, the mechanism by which these variants increase risk of a disease, for example the genes that are affected by such a variant, are often not known.

\subsection{Understanding the effects of rare variants on gene expression}
As discussed in \textbf{chapter 2}, the effect sizes of established GWAS risk variants are usually higher for rare, low frequency variants. Because eQTL and meQTL studies are of substantially lower sample size than GWAS studies, it is difficult to assess the effects of rare genetic variants on gene expression and methylation levels. Generally, these studies can only be conducted for common variants (minor allele frequency (MAF) $>$ 0.05), because in order to study low frequency variants large numbers of samples are required, which is expensive to realize. However, in yeast it has been shown that rare variants can have strong effects on gene expression, often more than common variants\cite{bloomRareVariantsContribute2019}. A study in humans of 472 genes showed that both very high and very low expressed genes had a high burden of rare variants\cite{zhaoBurdenRareVariants2016}, showing that rare variants have strong effects on gene expression in humans as well. Evidently, identifying the effects of rare variants on gene expression is important for understanding the genetic mechanism of disease. Others have shown allele specific expression (ASE) effects in many different tissues\cite{castelVastResourceAllelic2020} and that it can help with diagnosing rare disease patients\cite{mohammadiGeneticRegulatoryVariation2019a}. In \textbf{chapter 3}, we performed allele specific expression in a BIOS, a large Dutch healthy population cohort with RNA-seq measurements from blood. Typically, when interested in rare variants, genotypes from whole genome sequencing (WGS) are preferable over chip-array genotypes. Although it is possible to impute rare variants from chip genotypes, this is difficult without adding additional datasets to existing reference panels\cite{hoffmannStrategiesImputingAnalyzing2015}. Unfortunately, WGS data was only available for a subset of the BIOS samples. To still be able to assess the effects of rare variants on gene expression, we used the RNA-seq data to call genotypes as was done previously by Deelen \emph{et al.}\cite{deelenCallingGenotypesPublic2015b}. Additionally, we phased the RNA-seq based genotypes, which had not been done before with RNA-seq based genotypes. By phasing SNPs it is possible determine which alleles are one the same haplotype, which can improve allele specific expression measurements since sequence read counts that overlap multiple SNPs can then be combined. We used the WGS data of the subset of samples to validate both the genotypes and the phasing. Unfortunately, due to the fact that SNPs can only be called using RNA-seq data for genes that are actually expressed, phased haplotypes were often small. Because of this, we could get a better estimate of the allelic expression, as we could combine counts from the same haplotype, but we were unable to phase variants outside expressed genes and therefore could not assess the effects of regulatory regions on ASE.

Using this ASE approach, we were able to assess the effects of rare variants on gene expression levels. We found 5 individuals with predicted high impact heterozygous variants in known autosomal dominant disease genes. When assessing the allelic imbalance of these SNPs, we found that the reference allele was always much higher expressed than the alternative allele, and thanks to that the gene expression levels were stable in these samples. One hypothesis of how this could happen is due to a positive feedback loop: Let's say there is a gene A and gene B. The expression of gene A is regulated by gene B, and higher expression of gene A causes gene B to down regulate gene A. In normal circumstances, both alleles of gene A produce a functional transcript and the expression of both alleles is about equal. However, if one of the alleles of gene A has e.g. a stop-codon variant, the transcript from that allele is degraded due to nonsense-mediated decay. Now, there is less gene A product, and therefore gene B stops down regulating gene A. Gene A starts producing more transcripts, but due to nonsense mediated decay, only one allele produces a transcriptional product. When gene A and gene B are in homeostasis, one of the alleles is much higher expressed than the other. This is in line with the fact that we see an enrichment in ASE for genes containing stop-coding variants.

\subsection{Cellular and tissue context}
To find causal links and to understand the variant mechanism, it is imperative to study effects in the right tissue and cell type. This is one of the main drivers behind developing Decon2 in \textbf{chapter 4}, which allows us to deconvolute cell type interacting eQTLs from bulk RNA-seq samples. Although we could show our method to work well, it has some limitations: although we compare favourably against other cell count prediction tools xCell\cite{aranXCellDigitallyPortraying2017} and CIBERSORT\cite{newmanRobustEnumerationCell2015}, there is still an error in predicting cell count proportions based on gene expression. This error propagates when deconvoluting the cell type mediated eQTL effect from bulk tissue samples. Secondly, because we use cell count proportions there are significant correlations between cell types. These correlations makes it difficult to pinpoint the cell type specific effect. For example, if neutrophils and monocytes are anti-correlated to each other, and if an interaction effect is found in a neutrophils, it could also be that the effect is actually due to an opposite effect in monocytes. Thirdly, it is difficult to deconvolute the effect in specific sub-cell types, and for Decon2 we deconvoluted only the six main immune cell types. However, there are many more sub cell types that might be very relevant for understanding disease.

One method to overcome these three problems is to use single-cell RNA-seq\cite{tangMRNASeqWholetranscriptomeAnalysis2009}. Recently, this technique has been shown to be able to identify cell-type specific eQTLs\cite{vanderwijstSinglecellRNASequencing2018b}. Cell types can be assigned to single cells on the basis of expression levels of marker genes, removing the error of predicting cell counts. Because eQTLs are calculated within only those cells from a specific cell type, there is no problem of correlated cell count proportions. Thirdly eQTLs can be assessed even in rare cell types (although enough cells need to be available to have sufficient statistical power to detect eQTLs).

To further investigate the importance of cellular composition on eQTLs we compared \emph{cis}-eQTLs of different brain regions between each other and with blood \emph{cis}-eQTLs in \textbf{chapter 4}. We observed that cortical and cerebral regions of the brain show highly concordant effects. More differences were observed when contrasting these brain regions with the cerebellum, something that was observed in  GTEx as well\cite{GTExConsortiumAtlas}. Differences in genetic regulation of gene expression in brain cortex versus blood was often observed, although allelic concordance was high ($>$ 88\%).

\subsection{The need for more diversity in sampling}
One major obstacle of transitioning to precision medicine is that most of the large genetic cohort studies have been performed in European (52\%) or Asian (21\%) populations\cite{sirugoMissingDiversityHuman2019}. This is mainly problematic for polygenic disorders where there the causative variants and tagging SNPs are in weak LD\cite{sirugoMissingDiversityHuman2019}. For example, a multi-ethnic meta-analysis for pulmonary function identified over 50 new loci when including individuals from African, Asian, and Hispanic/Latino ethnicities\cite{wyssMultiethnicMetaanalysisIdentifies2018}.  Additionally, certain SNPs may only be present in specific populations\cite{adalsteinsdottirberglindNationwideStudyHypertrophic2014}.  In \textbf{chapter 5}, sufficient brain cortex samples from individuals from African and East Asian descent were available, permitting eQTL analysis in these populations. Surprisingly, we observed very strong concordance and directionality of eQTL effects across European, African and East Asian populations. This suggests that generally, genetic effects on gene expression levels are shared across populations, but that some of these are population specific due to allele frequency differences.

%\subsection{Drug predictions}
%One application of genetic regulation of gene expression not addressed in this thesis is the prioritisation of drug compounds. One of the tools that can be used for drug discovery is the connectivity map\cite{lambConnectivityMapUsing2006}. It has measured gene expression levels in cell cultures that were treated by different drug compounds, and compared that to gene expression levels of cell cultures that were not treated by any drug compounds Fisetin


\section{Future work}
Looking back, the transformation that the genomics and genetics fields have made over the last 20 years is amazing. There have been tremendous advances that have led to a much better understanding of the genetic basis of various diseases. These insights are now often driving translational research. For example, for the drug development pipeline, the number drug mechanisms with direct genetic support increases from 2\% at the preclinical stage to 8.2\% for approved drugs\cite{nelsonSupportHumanGenetic2015d}. Since for many diseases we know which variants are associated with increased risk thanks to the many large scale GWAS studies that have been done (currently the GWAS catalog\cite{macarthurNewNHGRIEBICatalog2017a} contains 4118 studies for 3418 traits), this information can be leveraged to improve drug development. However, to be able to do this in the most optimal way we need to understand the mechanism by which these variants drive disease risk, and to link them to causal genes and proteins.

With continued efforts in building comprehensive maps of genotype-phenotype relationships under various circumstances, further advancements in high throughput functional variant screenings such as with crispr-cas, and continued application of new insights from these efforts for new therapeutic options, we will see genetics play more and more important roles in healthcare and drug development.

Altogether, future studies aiming to improve our understanding of molecular effects of genetics need to combine a lot of different aspects. Ideally, they should be done with multi-ethnic individuals. If interested in a specific disease, this can be limited to populations with high prevalence. They should be conducted using single-cell technologies, as the cellular context of genetic effects is very important. With large\cite{vosaUnravelingPolygenicArchitecture2018c} and multi-tissue\cite{GTExConsortiumAtlas} studies having been completed with bulk data, findings from the single-cell studies can be replicated in bulk tissues if needed. Since physiology also plays an important role, the most informative studies would include both cases and healthy individuals. In case of e.g. an immune disease, stimulations should also be included. Finally, although this thesis focuses on genetic effects on gene expression, gene expression alone will not be able to capture the full cellular response. Gene expression and protein levels are reasonably but not perfectly correlated\cite{buccitelliMRNAsProteinsEmerging2020}. Additionally, SNPs might have effect on chromatin state or methylation but not on gene expression directly. Therefore, measurements should include not only RNA-seq, but also proteomics, ATAC-seq or DNASEseq.

\subsection{Collaboration and open data}
Finally, a lot of the work in this thesis, and all of \text{chapter 5}, is built on publicly available data. It has to be highlighted that without the academic community being so open to share their data, much of the genetic work discussed here would not be possible. It is great to see so many large scale academic collaborations, such as eQTLgen\cite{vosaUnravelingPolygenicArchitecture2018c} and GTEx\cite{GTExConsortiumAtlas}, pre-competitive public-private collaborations, such as OpenTargets, the UKBiobank\cite{sudlowUKBiobankOpen2015a} and Fingenn\cite{fingennFinnGenDocumentationR32020}, as well as large disease specific collaborations like Psychencode\cite{DbGaPStudy,ngXQTLMapIntegrates2017,wangComprehensiveFunctionalGenomic2018} and AMP-AD\cite{hodesAcceleratingMedicinesPartnership2016}.


\bibliographystyle{naturemag}
\bibliography{chapters/chapter6-discussion/chapter6-discussion}