\chapter{Discussion and Perspectives}
\label{chap:discussion}

{ \Large \leftwatermark{
\put(-67,-66.5){ 1 }
\put(-67,-91.5){ 2 }
\put(-67,-116.5){ 3 }
\put(-67,-141.5){ 4 }
\put(-67,-166.5){ 5 }
\put(-76.5,-199){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_wongOrange.eps}} \put(-67,-191.5){ {\color{white} 6 }}
} \rightwatermark{
\put(350.5,-66.5){ 1 }
\put(350.5,-91.5){ 2 }
\put(350.5,-116.5){ 3 }
\put(350.5,-141.5){ 4 }
\put(350.5,-166.5){ 5 }
\put(346.5,-199){\includegraphics[scale=0.8]{img/thumbindex/thumbindex_wongOrange.eps}} \put(350.5,-191.5){ {\color{white} 6 }}
}}

\newpage

\section{Summary of this thesis}
As mentioned in my introduction, the ultimate goal of most (human) genetics research is to improve healthcare outcomes. This can be done in many different ways, for example through drug discovery, biomarker identification, or better diagnosing, but also by improving our understanding of health and disease\cite{mcguireRoadAheadGenetics2020, claussnitzerBriefHistoryHuman2020e}. This thesis focusses on the latter, improving our understanding of health and disease. 

The regulatory networks in which genes form pathways are incredibly complex, with many layers of regulation. When trying to understand the function of a gene or the potential of a gene to be a candidate drug target, it is important to know how this gene functions within these complicated pathways. One way to approach this is by identifying which genetic factors affect the genes through eQTL studies, and subsequently building gene correlation networks to identify which pathways these genes are involved in. Preferably this should be done in cell type or tissue that is relevant for the phenotype of interest. In this thesis we have outlined the genetic architecture of molecular traits in \textbf{Chapter 1}\cite{claringbouldGeneticArchitectureMolecular2017} and identified the effect of rare variants on gene expression data in whole blood in \textbf{Chapter 2}\cite{kleinImbalancedExpressionPredicted2020}. We have developed statistical methods to look at cell type interacting genetic effects in blood in \textbf{Chapter 4}\cite{raulaguirre-gamboaDeconvolutionBulkBlood2020}, and we have analysed the effects of common variants on gene expression in the brain in Chapter 5. 

I will now discuss how this data and these methods can be used to help prioritize candidate causal genes, assist in assessing the feasibility of drug compounds and create hypotheses for the role of identified genes on disease phenotypes, and how future work can improve on this.

\section{Understanding genetic regulation of gene expression}
Genome-wide association studies (GWAS) have allowed us to gain valuable insights into the genomic loci that convey disease risk.  Thanks to the sharing of genetic information and GWAS summary statistics there are now many hundreds of datasets with disease associated variants available\cite{visscher10YearsGWAS2017}. However, the mechanism by which these variants heighten risk of a disease, for example the genes that are affected by such a variant, are often not known. 

\subsection{Understanding the effects of rare variants on gene expression}
As discussed in \textbf{chapter 2}, the effect sizes of GWAS variants become stronger for rare variants. Because eQTL and meQTL studies are of substantially lower sample size than GWAS studies, it is difficult to assess the effects of the rare variants on expression and methylation. Generally, these studies are done on common variants (minor allele frequency (MAF) > 0.05), as sample sizes are smaller than those for GWAS studies due to higher sequencing costs. Even so, in yeast it has been shown that rare variants have a stronger effect on gene expression than common variants\cite{bloomRareVariantsContribute2019}. A study in humans of 472 genes showed that very high and very low expressed genes had a high burden of rare variants\cite{zhaoBurdenRareVariants2016}. Evidently, identifying the effects of rare variants on gene expression is important for understanding the genetic mechanism of disease. Others have shown ASE effects in many different tissues<cite gtex> and that it can help with diagnosing<cite the muscle ase paper>, but generally these are done with limited number of samples or limited number of genes. In \textbf{chapter 3}, we performed allele specific expression in a BIOS, a large Dutch healthy population cohort with RNA-seq measurements from blood. Typically, when looking at rare variants genotypes from whole genome sequencing are preferable over chip-array genotypes, as the chip-array typically contain only common variants, and imputing rare variants is difficult<citation?>. Unfortunately, whole genome sequence data was not available. Instead, we used the RNA-seq data to do genotype calling as was done previously by Deelen \emph{et al.}<citation>. Additionally, we phased the RNA-seq based genotypes, something not done with RNA-seq based genotypes before. Phasing SNPs gives information of which alleles are on the same haplotypes, which can improve allele specific expression measurements as counts from multiple SNPs can be combined. Unfortunately, due to the fact that SNPs called with RNA-seq are only found in regions with expressed exons, the phased haplotypes were often small. 

Using this approach we were able to assess the effects of rare variants on gene expression, and we found 5 individuals with predicted high impact heterozygous variants in known autosomal dominant disease genes. It was surprising to find these variants, as the participants had no known genetic disorders. When assessing the allelic imbalance of these SNPs, we found that the reference allele was always much higher <fill in minimum ratio> expressed than the alternative allele. One hypothesis of how this could happen due to a positive feedback loop. <REWRITE FROM HERE>if a gene with e.g. a stop-codon variant transcript product gets degraded due to nonsense-mediated decay. If there is a downstream gene that.... We see an enrichment in ASE for stop-coding variants.

This information can help in diagnosis by...


\subsection{Digging into cellular contxt}
To find causal links and to understand the variant mechanism, it is imperative to study effects in the right tissue and cell type. This is one of the main drivers behind developing Decon2 in \textbf{chapter 4}, which allows us to deconvolute cell type interacting eQTLs from bulk RNA-seq samples. However, there are some limitations to this approach. For one, although we compare favourably against other cell count prediction tools xCell\cite{aranXCellDigitallyPortraying2017} and CIBERSORT\cite{newmanRobustEnumerationCell2015}, there is still an error in the predicted cell count proportions. This propagates to the eQTL deconvolution. Secondly, because we use cell count proportions there are significant correlations between cell types. This makes it difficult to pinpoint the cell type specific effect. For example, if neutrophils and monocytes are anti-correlated to each other, and if an interaction effect is found in Neutrophil, it could also be that the effect is actually due to an opposite effect in monocytes. Thirdly, it is difficult to deconvolute the effect in sub-cell types, and for Decon2 we deconvoluted only the 6 main immune cell types. However, there are many more sub cell types that might be very relevant for understanding disease.

One method to overcome these three problems is to use single-cell RNA-seq\cite{tangMRNASeqWholetranscriptomeAnalysis2009}. Recently, this technique has been shown to be able to identify cell-type specific eQTLs\cite{vanderwijstSinglecellRNASequencing2018b}. Cell types can be assigned to single cells on the basis of expression levels of marker genes, removing the error of predicting cell counts. Because eQTLs are calculated within only those cells from a specific cell type, there is no problem of correlated cell count proportions. Thirdly eQTLs can be assessed even in rare cell types (although enough cells need to be available to have enough power to detect eQTLs).

\subsection{Disease dependent effects}
Genetic effects are not only dependent on the cellular context, but physiological context as well. 

\subsection{Functional validation}
This work has been ... crispr

\subsection{ethnicity}

\subsection{Drug predictions}


\section{Final thoughts}
Reading back the transformation that the genomics and genetics fields have made over the last 20 years is amazing. There have been tremendous advances in the last 20 years that have lead to a much better understanding of the genetic basis of diseases. These insights are now often driving translational research. For example for the the drug development pipeline, drug mechanism with direct genetic support increases from 2\% at the preclinical stage to 8.2\% for approved drugs\cite{nelsonSupportHumanGenetic2015d}. Since for many diseases we know which variants are associated with increased risk thanks to the many large scale GWAS studies that have been done (currently the GWAS catalog\cite{macarthurNewNHGRIEBICatalog2017a} contains 4118 studies for 3418 traits), this information can be leveraged to improve drug development. However, to be able to do this in the most optimal way we need to understand the mechanism by which these variants drive disease risk, and to link them to causal genes and proteins.
 With continued efforts in building comprehensive maps of genotype-phenotype relationships under various circumstances, further advancements in high throughput functional variant screenings such as with crispr-cas, and continued application of new insights from these efforts for new therapeutic options, we will see genetics play more and more important roles in healthcare and drug development. 

Finally, a lot of the work in this thesis, and all of \text{chapter 5}, is built on publicly available data. Without the academic community being so open to sharing their data<finish sentence> 
